crates/router/src/connector/shift4/transformers.rs

```diff
@@ -286,10 +282,9 @@ impl TryFrom<&api_models::payments::GiftCardData> for Shift4PaymentMethod {
     type Error = Error;
     fn try_from(gift_card_data: &api_models::payments::GiftCardData) -> Result<Self, Self::Error> {
         match gift_card_data {
-            payments::GiftCardData::Givex(_) => Err(errors::ConnectorError::NotSupported {
-                message: utils::SELECTED_PAYMENT_METHOD.to_string(),
-                connector: "Shift4",
-            }
+            payments::GiftCardData::Givex(_) => Err(errors::ConnectorError::NotImplemented(
```

Comment:
Here, as `givex` and `paysafecard` are throwing the same error now.
Could you please OR ( | ) between them and avoid the repetition of same error.